docs/unity/part-1.md

```diff
@@ -639,21 +649,26 @@ using SpacetimeDB;
 private float? lastUpdateTime;
 private void FixedUpdate()
 {
-    if ((lastUpdateTime.HasValue && Time.time - lastUpdateTime.Value > 1.0f / movementUpdateSpeed) || !SpacetimeDBClient.instance.IsConnected())
-    {
-        return;
-    }
-
-    lastUpdateTime = Time.time;
-    var p = PlayerMovementController.Local.GetModelPosition();
-    Reducer.UpdatePlayerPosition(new StdbVector3
-        {
-            X = p.x,
-            Y = p.y,
-            Z = p.z,
-        },
-        PlayerMovementController.Local.GetModelRotation(),
-        PlayerMovementController.Local.IsMoving());
+   bool hasUpdatedRecently = lastUpdateTime.HasValue && 
+   Time.time - lastUpdateTime.Value > 1.0f / movementUpdateSpeed;
```

Comment:
I like breaking up these variables :100: 

..but this line break/indent is super confusing :sweat_smile:

Reply:
also, you didn't introduce this issue, but since we're here: this code is wrong!

Maybe we can add another variable?

```rust
   
   float? deltaTime = Time.time - lastUpdateTime;
   bool hasUpdatedRecently = deltaTime.HasValue && deltaTime.Value < 1.0f / movementUpdateSpeed;
   bool isConnected = SpacetimeDBClient.instance.IsConnected();

   if (hasUpdatedRecently || !isConnected)
   {
      return;
   }
```

Reply:
Got it -- also, I'm wondering how you folks feel about this:

```csharp
 if (hasUpdatedRecently || !isConnected)
      return;
```

Do you like the shortcut versions for simple `if`s? I didn't touch these, just in case.

Reply:
I'm indifferent - @jdetter does it matter to you?